_includes/components/header.html

```diff
@@ -7,7 +7,9 @@
     <button class="usa-menu-btn">Menu</button>
     <div class="usa-logo" id="logo">
       <em class="usa-logo-text">
-        <a href="{{ site.baseurl }}" accesskey="1" title="Home" aria-label="Home">{{ header.title | default: site.title }}</a>
+        <a href="{{ header.href | default: site.baseurl }}"
+          title="Home" aria-label="Home">{{ header.title | default: site.title }}
```

Comment:
I don't know that `title="Home" aria-label="Home"` is needed. Also see: https://github.com/18F/web-design-standards/issues/1648